internal/ignition/ignition.go

```diff
@@ -155,11 +155,16 @@ create_template_file
 correlate_int_mac
 `
 
+const (
+	masterIgn = "master.ign"
```

Comment:
bootstrap.ign?

Reply:
next time ? :) 
wanted to consts only those i actually touch :)